users/faq-parts/usage.rst

```diff
@@ -87,6 +87,17 @@ block) from A, and then as A gets rescanned remove the files from A.
 A workaround would be to copy first from A to B, rescan B, wait for B to
 rebuild on remote ends, and then delete from A.
 
+Can I help initial sync by copying files manually?
+--------------------------------------------------
+
+If you have a large folder that you want to keep in sync over a not-so-fast network, and you have the possibility to move all files to the remote site in a faster manner, here is a procedure to follow:
+
+- Create the folder on site 1, but don't share it with anyone.
+- Copy the files from site 1 to site 2 using normal file copy. If this takes a long time (perhaps travelling to site 2), it could be a good idea to make sure that the files on site 1 is not updated while you do this.
```

Comment:
```suggestion
- Copy the files from device A to device B using regular file copy. If this takes a long time (perhaps requiring travelling to device B physically), it may be a good idea to make sure that the files on device A are not updated while you are doing this.
```